src/AdminConsole/Components/Pages/App/Reporting.razor

```diff
@@ -0,0 +1,67 @@
+@page "/app/{app}/Reporting"
```

Comment:
We don't need to modify the route in the middleware anymore. Do not rename {app}.